src/System.Security.Cryptography.Algorithms/tests/BlockSizeValueTests.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Security.Cryptography.Hashing.Algorithms.Tests
+{
+    public class BlockSizeValueTests
+    {
+        public static IEnumerable<object[]> GetBlockSizeValue()
+        {
+            return new[]
+            {
+                new object[] { new HMACMD5Test().GetBlockSizeValue(),    64  },
+                new object[] { new HMACSHA1Test().GetBlockSizeValue(),   64  },
+                new object[] { new HMACSHA256Test().GetBlockSizeValue(), 64  },
+                new object[] { new HMACSHA384Test().GetBlockSizeValue(), 128 },
+                new object[] { new HMACSHA512Test().GetBlockSizeValue(), 128 },
```

Comment:
I just realized that this is creating-then-ignoring a bunch of Disposable objects.  It'd be nice if they got cleaned up, since they are ultimately holding finalizable objects.